exercises/angular/2-building-first-app/problem/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import 'zone.js'; // Work around issue in StackBlitz `Error: NG0908: In this configuration Angular requires Zone.js`
```

Comment:
Almost all of the code is exactly as it was when generated, but I did have to add this for StackBlitz.

Reply:
yeah it seems it is needed for stackblitz, but it was also added. for codesandbox, is it needed there as well?

Reply:
I don’t want us to maintain multiple versions of the same step, so we’ll have one that covers everything we support.